app/services/waste_carriers_engine/notify/registration_confirmation_letter_service.rb

```diff
@@ -39,7 +39,7 @@ def base_personalisation
         {
           contact_name: contact_name,
           reg_identifier: @registration.reg_identifier,
-          company_name: @registration.company_name,
+          business_name: @registration.business_name,
```

Comment:
Notify template would need updating to support this parameter

Reply:
Following discussion - too many potential downstream impacts, reverting the database attribute name change.